### DISCUSSION TITLE
Sonic Pad

### DISCUSSION THREAD

**Discussion by Fausti40** [CrealityOfficial/Creality_Sonic_Pad#16] [Q&A]
Hallo würde mir gerne das Sonic Pad kaufen,habe aber das bigtreetech-skr-mini -e3-v2.0 habe auch schon die cfg datei gefunden.Aber wie kann ich sie im Sonic Pad einbinden (Habe ein Ender 3E)bin für jede Hilfe dankbar.

**Comment by kamoteshake** [comment]
Based on this [Reddit post](https://www.reddit.com/r/klippers/comments/yr2y6w/skr_mini_e3_v3_with_sonic_pad_and_klipper/), it sounds like it is difficult to install Klipper using the Sonic pad to any board other than Creality's. Unless you have other Creality printers that are supported by Sonic Pad, I would just try and install Klipper on your own.
